htdocs/pages/forumphp2015/inscription.php

```diff
@@ -1,6 +1,6 @@
 <?php
 
-require_once '../../include/prepend.inc.php';
+require_once dirname(__FILE__) . '/../../include/prepend.inc.php';

```

Comment:
Just for my own information, we runs PHP >= 5.3 right?

In this case it should be better to use `__DIR__` instead of `dirname(__FILE__)`.

Signed: a friend